bears/js/ESLintBear.py

```diff
@@ -58,15 +58,28 @@ def generate_config(filename, file):
         return '{"extends": "eslint:recommended"}'
 
     def process_output(self, output, filename, file):
+        # Taking output from stderr for ESLint 2 program errors.
+        # ESLint 2 is no longer supported, but it is here so that anyone
+        # with ESLint 2 will still see any errors that were emitted.
         if output[1]:
+            output = output[1]
+        else:
+            output = output[0]
```

Comment:
but we require ESLint 3 now, don't we?

Reply:
Do you mean ESLint 2 ?

Unfortunately we need ESLint 2 just to test this block :(

Reply:
See @jayvdb review

Reply:
@blazeu , this is a good case to use `# pragma: no cover` (on [the branch](http://coverage.readthedocs.io/en/coverage-4.3.4/excluding.html) which will not be covered by the tests.)

Reply:
> But, unfortunately we need ESLint 2 just to test this block 

Yeah but I think because of ESLint 3 we don't need this block at all :O